linkedin/linkedin-connector/linkedin-connector-1.0.0/org.wso2.carbon.connector/src/main/resources/linkedin_people/getConnections.xml

```diff
@@ -35,7 +35,7 @@
 			if (id != null && id != "") {
 				query = apiUrl + 'id=' + mc.getProperty('uri.var.id') + '/connections';
 			} else if(publicProfileUrl != null && publicProfileUrl != "") {
-				var encoded_publicProfileUrl = encodeURIComponent(mc.getProperty('uri.var.publicProfileUrl'));
+				var encoded_publicProfileUrl = mc.getProperty('uri.var.publicProfileUrl');
```

Comment:
Recently we have rewritten the LinkedIn connector. The getConnections method is deprecated from the connector